main.go

```diff
@@ -39,63 +41,81 @@ import (
 	"github.com/sonatype-nexus-community/nancy/parse"
 )
 
+var appLog = logger.Logger
+
 func main() {
+	appLog.Debug("Starting Nancy")
+
 	if len(os.Args) > 1 && os.Args[1] == "iq" {
+		appLog.Debug("Nancy parsing config for IQ")
 		config, err := configuration.ParseIQ(os.Args[2:])
 		if err != nil {
 			flag.Usage()
 			os.Exit(1)
 		}
 		processIQConfig(config)
+		appLog.Debug("Nancy finished parsing config for IQ")
 	} else {
+		appLog.Debug("Nancy parsing config for OSS Index")
 		ossIndexConfig, err := configuration.Parse(os.Args[1:])
 		if err != nil {
 			flag.Usage()
 			os.Exit(1)
 		}
 		processConfig(ossIndexConfig)
+		appLog.Debug("Nancy finished parsing config for OSS Index")
 	}
 }
 
 func printHeader(print bool) {
 	if print {
+		appLog.Debug("Attempting to print header")
 		figure.NewFigure("Nancy", "larry3d", true).Print()
 		figure.NewFigure("By Sonatype & Friends", "pepper", true).Print()
 		log.Println("Nancy version: " + buildversion.BuildVersion)
+		appLog.Debug("Finished printing header")
```

Comment:
While we are hear might as well dump the current Nancy version into the log file as well so that you dont have to ask people for it you just have to ask for the log file.